active/DensityStyle/UseCompactResources.md

```diff
@@ -0,0 +1,47 @@
+# Background
+Compact feature is described in [Suggestion: Add Compact Dictionary to WinUI 2.1](https://github.com/Microsoft/microsoft-ui-xaml/issues/98).
+This provides API to enable Compact in application level.
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+UseCompactResources property is added XamlControlsResources to allow customer to enable Compact.
+
+# Examples
+
+```
+    runtimeclass XamlControlsResources : Windows.UI.Xaml.ResourceDictionary

```

Comment:
This isn't an example, this is the API Details. Could you move this down to that section?